server/resp/src/main/java/org/infinispan/server/resp/commands/cluster/SegmentSlotRelation.java

```diff
@@ -0,0 +1,57 @@
+package org.infinispan.server.resp.commands.cluster;
+
+import static org.infinispan.commons.logging.Log.CONFIG;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.infinispan.distribution.ch.impl.CRC16HashFunctionPartitioner;
+
+public class SegmentSlotRelation {
+
+   static final int SLOT_SIZE = 1 << 14;
```

Comment:
Put a comment here just so we know why this is hardcoded (ie. REDIS has it hard coded)